concent_api/core/tests/utils.py

```diff
@@ -22,6 +21,22 @@
 from golem_messages.factories.tasks import ReportComputedTaskFactory
 from golem_messages.factories.tasks import TaskToComputeFactory
 from golem_messages.message.base import Message
+from golem_messages.message.concents import ForceGetTaskResult
+from golem_messages.message.concents import ForcePayment
+from golem_messages.message.concents import ForceReportComputedTask
+from golem_messages.message.concents import ForceSubtaskResults
+from golem_messages.message.concents import ForceSubtaskResultsResponse
+from golem_messages.message.concents import SubtaskResultsVerify
+from golem_messages.message.tasks import AckReportComputedTask
+from golem_messages.message.tasks import CannotComputeTask
+from golem_messages.message.tasks import ComputeTaskDef
+from golem_messages.message.tasks import RejectReportComputedTask
+from golem_messages.message.tasks import ReportComputedTask
+from golem_messages.message.tasks import SubtaskResultsAccepted
+from golem_messages.message.tasks import SubtaskResultsRejected
+from golem_messages.message.tasks import TaskFailure
+from golem_messages.message.tasks import TaskMessage
+from golem_messages.message.tasks import TaskToCompute
```

Comment:
I think better way to do imports is to import just `from golem_messages import message`

Reply:
I did it like this, because this messages are multiple times used in code. I think it is better to import all of them like this and use in code like simple i.e. `SubtaskResultsRejected` instead of `message.tasks.SubtaskResultsRejected`. 
What do you think?

Reply:
So then better to import these two modules: 
`from golem_messages.messages import concents`
`from golem_messages.messages import tasks` 
It will be more readable I think :)

Reply:
I personally prefer the style used by Pawel here. The prefixes only add noise, especially if you use them often.

It's the same thing we do in `models` - we import each thing separately instead of just importing `django.db.models` because it makes model definitions more readable.